src/socket/udp.rs

```diff
@@ -8,20 +8,27 @@ use crate::socket::PollAt;
 #[cfg(feature = "async")]
 use crate::socket::WakerRegistration;
 use crate::storage::Empty;
-use crate::wire::{IpEndpoint, IpListenEndpoint, IpProtocol, IpRepr, UdpRepr};
+use crate::wire::{IpEndpoint, ip::Address, IpListenEndpoint, IpProtocol, IpRepr, UdpRepr};
```

Comment:
You could just import `IpAddress` instead of `ip::Address`.

Reply:
Changed, thanks.